pkgs/tools/filesystems/e2fsprogs/default.nix

```diff
@@ -14,6 +14,10 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ pkgconfig texinfo ];
   buildInputs = [ libuuid gettext ];
 
+  preConfigure = stdenv.lib.optionalString stdenv.hostPlatform.isMusl ''
+    NIX_CFLAGS_COMPILE+=" -D__GNUC_PREREQ(maj,min)=0"
```

Comment:
Can you add a short explanation here? It looks quite cryptic.

Reply:
This can be also done in nix no?
```nix
NIX_CFLAGS_COMPILE = stdenv.lib.optional stdenv.hostPlatform.isMusl " -D__GNUC_PREREQ(maj,min)=0";
```

Reply:
Good call, updated thank you! How's it look now? :)